crates/boundless-cli/src/config.rs

```diff
@@ -175,8 +175,8 @@ impl ProverConfig {
         if let Some(ref api_key) = self.bento_api_key {
             std::env::set_var("BONSAI_API_KEY", api_key);
         } else {
-            tracing::debug!("No API key provided. Setting BONSAI_API_KEY to empty string");
-            std::env::set_var("BONSAI_API_KEY", "");
+            tracing::debug!("No API key provided. Setting BONSAI_API_KEY to reserved:1");
+            std::env::set_var("BONSAI_API_KEY", "reserved:1");
```

Comment:
This probably doesn't matter, but before, by default, the value was 0, so maybe its worth maintaining that behaviour?